app/views/tasks/show.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#todoDisplayModal').html('<%= escape_javascript render("show") %>');
```

Comment:
`escape_javascript` has an alias of `j`, which you might see out in the wild. You can use whichever you prefer.

Reply:
Thanks for the tip!